README.rst

```diff
@@ -23,6 +23,15 @@ Dependencies
 * `jsonfield <https://github.com/rpkilby/jsonfield>`_
 
 
+Optional Dependency
+-------------------
+
+* `lxml <https://lxml.de/>`_
+
+Add this optional dependency, if you don't trust the HTML beeing rendered inside the Django admin
+backend. This library then removes potentially dangerous tags in order to prevent XSS attacks.
```

Comment:
I think insecure by default is not the right approach here. I'd rather the html body is rendered *only* if lxml is installed. If it isn't, just show a message saying `Install lxml to see HTML preview`.

Reply:
After examining which HTML tags, attributes and styles are allowed within emails, I came up with a complete configuration for `bleach`.
Without `bleach`, HTML emails are rendered without tags, using Django's internal `strip_tags`. This setup should be safe regardless of its optional dependencies.